app/assess/templates/sub_criteria.html

```diff
@@ -68,6 +69,22 @@ <h2 class="govuk-heading-m">Comments</h2>
                         {% endif %}
                     {% endfor %}
                     {% endif %}
+                    
+                    {% if displayCommentBox != True %}
+                    <form method="GET">
+                        <button id="comment-button-community" class="govuk-button secondary-button govuk-!-margin-top-2" type="submit" data-module="govuk-button" value="add-comment">
+                            {% if comments == None %} Add a comment
+                            {% else %} Add another comment
+                            {% endif %}
+                        </button>
+                        <input type="hidden" name="theme_id" value="{{current_theme_id}}">
+                        <input type="hidden" name="add-comment" value="1">
```

Comment:
Is this hidden value only used for initially hiding then showing the comment box?

If it's a boolean, we could use a checkbox instead of a pseudo bool, but I'm not too fussed tbh as it's only a minor difference